src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -101,6 +123,88 @@ public void resetOnPath(String path) {
         // Intentionally blank
     }
 
+    /**
+     * Helper method which handles the file creation for both normal file uploads and for markdown
+     * @param file The file to be uploaded
+     * @param keepFileName specifies if original file name should be kept
+     * @param markdown boolean which is set to true, when we are uploading a file within the markdown editor
+     * @return The path of the file
+     */
+    @NotNull
+    public String handleSaveFile(MultipartFile file, boolean keepFileName, boolean markdown) {
+        // NOTE: Maximum file size is set in resources/config/application.yml
+        // Currently set to 10 MB
```

Comment:
```suggestion
```
The concrete value is not important for any of the code in this file. Removing the remark removes one potential inconsistency between code and comment.